examples/todoapp/src/generatedNoCheck/internal/@foundry/ontology-runtime/dist/baseTypes/geoshapes/DistanceMethods.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  ofCentimeters as OG_ofCentimeters,
```

Comment:
We probably don't need to export these and can just export distance, since they're a bit strange by themselves (unless there's a reason for this)

Reply:
If the treehsaking is a problem we dont need to make it better for 1.1

Reply:
```
.error TS2742: The inferred type of 'Distance' cannot be named without a reference to '$path'. This is likely not portable. A type annotation is necessary.
```